packages/create-analog/template-blog/src/app/pages/blog/index.page.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import { injectContentFiles } from '@analogjs/content';
+import PostAttributes from '../../post-attributes';
+
+@Component({
+  selector: 'app-blog',
+  standalone: true,
+  template: `
+    <h1>Blog Archive</h1>
+    @for (post of posts; track post.attributes.slug) {
+    <a [href]="'/blog/' + post.attributes.slug">
```

Comment:
```suggestion
    <a [routerLink]="['/blog/', post.attributes.slug]">
```

Reply:
Pushed a change to resolve this (as well as the next comment)